karma.conf.js

```diff
@@ -0,0 +1,42 @@
+module.exports = function (config) {
+    config.set({
+        frameworks: ['jasmine', 'karma-typescript'],
+        files: [
+            {
+                pattern: 'src/**/*.ts'
+            },
+            {
+                pattern: 'spec/**/*.ts'
+            }
+        ],
+        plugins: [
+            'karma-jasmine',
+            'karma-typescript',
+            'karma-coverage',
```

Comment:
Isn't this quite the duplicate? `karma-coverage` & `karma-coverage-istanbul-reporter`